File: N1.5/N1.5.js
function StringFixer(Line, Globalcount) {
    let WorkRow = Line;
    let count = 0;
    for (let i = 0; i < WorkRow.length - 1; i++) {
        let code_symbol = WorkRow.charCodeAt(i);
        code_symbol++;
        let next_symbol = String.fromCharCode(code_symbol);
        if (WorkRow[i] === WorkRow[i + 1]) {
            count++;
        }
        if (count === 2) {
            WorkRow = WorkRow.substring(0, i + 1) + next_symbol + WorkRow.substring(i + 1);
            count = 0;
            Globalcount[0]++;
        }
    }
    return WorkRow;
}

function main() {
    const Globalcount = [0];
    console.log("Enter row, please - ");
    const readline = require('readline');
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });

    rl.question('', (Line) => {
        const FinalRow = StringFixer(Line, Globalcount);
        console.log("\nFixed row - " + FinalRow + "\nValue of changes - " + Globalcount[0]);
        rl.close();
    });
}

main();

File: N1.5/N1.5.go
package main

import (
	"fmt"
	"strings"
)

func StringFixer(Line string) (string, int) {
	var WorkRow strings.Builder
	count := 0
	Globalcount := 0

	for i := 0; i < len(Line)-1; i++ {
		codeSymbol := int(Line[i])
		codeSymbol++
		nextSymbol := byte(codeSymbol)
		WorkRow.WriteByte(Line[i])

		if Line[i] == Line[i+1] {
			count++
		}

		if count == 2 {
			WorkRow.WriteByte(nextSymbol)
			count = 0
			Globalcount++
		}
	}
	WorkRow.WriteByte(Line[len(Line)-1])

	return WorkRow.String(), Globalcount
}

func main() {
	var Globalcount int
	var Line string

	fmt.Print("Enter row, please - ")
	fmt.Scanln(&Line)

	FinalRow, Globalcount := StringFixer(Line)

	fmt.Printf("\nFixed row - %s\nValue of changes - %d", FinalRow, Globalcount)
}
